keras/layers/convolutional.py

```diff
@@ -3409,6 +3409,12 @@ def compute_output_shape(self, input_shape):
     return tf.TensorShape([input_shape[0], length, input_shape[2]])
 
   def call(self, inputs):
+    if tf.not_equal(tf.size(inputs), 0) and sum(self.cropping) >= inputs.shape[1]:
+      raise ValueError(
+        'cropping parameter of Cropping layer is too high,' +
```

Comment:
This would read clearer with format strings, and we are trying to gravitate towards more uniform error messages in keras.

f'`cropping` parameter of Cropping layer must be greater than the input shape. '
f'Recieved: inputs.shape={inputs.shape}, and cropping={self.cropping}'

Reply:
Did it both changes!